liteflow-labs/starter-kit
# Fix / Token card overflow issue

### Project organization

- Closes #177 

### Description

Fixes various overflow issues regarding Token card component.

### How to test

On inspect mode choose Samsung Galaxy S8+ (360px wide) to check on lowest width possible

### Checklist


### Note to reviewer

I believe the issue with auction overflow was fixed with image size PR. Couldn't reproduce. Let me know if it still exists and provide me how to reproduce please.
